index.js

```diff
@@ -60,4 +61,20 @@ async function fetchCharacters() {
   }
 }
 
+
+//Search Bar
+searchBar.addEventListener("submit", (event) => {
+  event.preventDefault();
+  const formData = new FormData(event.target);
+  const data = Object.fromEntries(formData);
+  searchQuery = data.query;
+  cardContainer.textContent = "";
+  fetchCharacters();
+  pageIndex = 1
```

Comment:
Wenn wir in Zeile 49 folgendes einfügen:
    const maxPage = data.info.pages
    pagination.textContent = `${pageIndex} / ${maxPage}`;
Wird die maximale Seitenzahl dynamisch aus der API gezogen